akka-actor-tests/src/test/scala/akka/util/ByteStringSpec.scala

```diff
@@ -885,7 +955,7 @@ class ByteStringSpec extends WordSpec with Matchers with Checkers {
             case (xs, from, until) =>
               likeVector(xs)({ it =>
                 val array = new Array[Byte](xs.length)
-                it.slice(from, until).copyToArray(array, from, until)
```

Comment:
not sure why this would slice it first before

Reply:
Yep, seems pretty wrong. Wouldn't the indices be wrong after having called slice before?

Reply:
The parameters for copyToArray is actually offset in target array, and lenght to copy, and this prop test is to verify that it does the same on vector and bytestring which it would do. But weird test.